backend/Makefile

```diff
@@ -12,24 +12,39 @@
 OUT_DIR = dist/
 DATA_DIR = data
 
-BUILD_ARGS_BASE = -p src --distpath $(OUT_DIR) -y --clean -F
+################################### Building ###################################
+################## Unix and Windows use different delimiters ###################
+################################################################################ 
+ifeq ($(OS),Windows_NT)  # Windows_NT, XP, 2000, 7, 8, 10, 11...
+	DELIMITER = ;
+else
+	DELIMITER = :
+endif
+BUILD_ARGS = -p src --distpath $(OUT_DIR) -y --clean -F --add-data $(DATA_DIR)$(DELIMITER)$(DATA_DIR)
 
-# UNIX like systems use ':' as the separator while Windows uses ';'
-BUILD_ARGS_UNIX = $(BUILD_ARGS_BASE) --add-data $(DATA_DIR):$(DATA_DIR)
-BUILD_ARGS_WIN = $(BUILD_ARGS_BASE) --add-data $(DATA_DIR);$(DATA_DIR)
-
-make: unix
-
-unix:
-	pyinstaller $(BUILD_ARGS_UNIX) src/BeyondRGB.py
-
-win:
-	pyinstaller $(BUILD_ARGS_WIN) src/BeyondRGB.py
+make: 
+	pyinstaller $(BUILD_ARGS) src/BeyondRGB.py
 
 clean:
 	rm BeyondRGB.spec dist build out -rf
+	
+###########################VENV#######################################
+	
+venv-init:
+	( \
+	 	python3 -m venv ./venv; \
+	 	. venv/bin/activate; \
```

Comment:
This command fails when run in the make file. It works when run manually for some reason though.